manager/integration/tests/test_node.py

```diff
@@ -350,6 +350,22 @@ def test_replica_scheduler_no_disks(client):  # NOQA
     common.wait_for_volume_delete(client, vol_name)
 
 
+@pytest.mark.node  # NOQA
+def test_disable_scheduling_on_cordoned_node(client):  # NOQA
+    """
+    Test replica scheduler: schedule replica based on
+    `Disable Scheduling On Cordoned Node` setting
+
+    1. Set `Disable Scheduling On Cordoned Node` to true.
+    2. Set `Replica Soft Anti-Affinity` to false.
+    3. Set cordon on one node.
+    4. Create a volume and verify the scheduler should fail due to cordoned node.
+    5. Remove step 4 volume.
```

Comment:
Don't need to remove the volume on step 4. Once the scheduling resumed, the volume should be scheduled correctly.